app/configs/crashesColumns.tsx

```diff
@@ -2,255 +2,259 @@ import { ColDataCardDef } from "@/types/types";
 import { Crash } from "@/types/crashes";
 import { formatDateTime } from "@/utils/formatters";
 
-export const crashesColumns: {
-  [name: string]: ColDataCardDef<Crash>;
-} = {
+export const crashesColumns = {
   active_school_zone_fl: {
-    name: "active_school_zone_fl",
+    path: "active_school_zone_fl",
     label: "Active school zone",
     editable: true,
     inputType: "yes_no",
   },
   address_primary: {
-    name: "address_primary",
+    path: "address_primary",
     label: "Address",
   },
   address_secondary: {
-    name: "address_secondary",
+    path: "address_secondary",
     label: "Secondary address",
   },
   at_intrsct_fl: {
-    name: "at_intrsct_fl",
+    path: "at_intrsct_fl",
     label: "At intersection",
     editable: true,
     inputType: "yes_no",
   },
   case_id: {
-    name: "case_id",
+    path: "case_id",
     label: "Case ID",
   },
   crash_speed_limit: {
-    name: "crash_speed_limit",
+    path: "crash_speed_limit",
     label: "Speed limit",
     editable: true,
     inputType: "number",
   },
   crash_timestamp: {
-    name: "crash_timestamp",
+    path: "crash_timestamp",
     label: "Crash date",
     valueFormatter: formatDateTime,
   },
-  fhe_collsn_id: {
-    name: "fhe_collsn_id",
+  collsn: {
+    path: "collsn.label",
     label: "Collision type",
     editable: true,
     inputType: "select",
-    relationshipName: "collsn",
-    lookupTable: {
+    relationship: {
       tableSchema: "lookups",
       tableName: "collsn",
+      idColumnName: "id",
+      labelColumnName: "label",
+      foreignKey: "fhe_collsn_id",
```

Comment:
here is the new `Path` type in action, along with `relationship` metadata. the very cool thing here is that the `path` property is type-checked against the `Crash` type. so `"collsn.label"` is valid but `collsn.pizza` would not be. 

so this new `Path` type unlocks adding column metadata for nested properties in a typesafe way 🌈